lib/theory/src/Theory/Tools/Wellformedness.hs

```diff
@@ -457,6 +476,64 @@ factReports thy = concat
         : (do RuleItem ru <- get thyItems thy; get rActs $ get oprRuleE ru)
           ++ (do RuleItem ru <- get thyItems thy; racs <- get oprRuleAC ru; get rActs racs)
 
+    
+    -- | To bound a fact with his most similar fact
+    -- | The editing distance of two facts must between 1(includ) and 3 (includ)
```

Comment:
Is this required or ensured by the function?

Reply:
It's only required by the function factLhsOccurNoRhs